modules/appeals_api/config/schemas/shared_schemas.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
I like the idea but we'll want each of these schemas to be a fully-realized schema file definition (with the `$schema` link & `definitions`), etc. You could potentially wrap the response with these things, or separate each of these schemas into their own files.

Also let's adjust the `$schema` definition to the latest instead of draft-7

Reply:
Done!